src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -94,7 +94,7 @@ private void OnEventSourceChanged()
 
                 // When something changes, clear the cached data we have.
                 this.CachedTags = null;
-                this.CachedSnapshot = null;
+                this.CachedTaggedSpan = default(SnapshotSpan);
```

Comment:
Should `CachedTaggedSpan` be nullable? I always get worried around `default(SnapshotSpan)` since that's invalid by the type.